inlang/source-code/sdk/src/ast.ts

```diff
@@ -0,0 +1,120 @@
+import { LanguageTag as Locale } from "@inlang/language-tag"
```

Comment:
@jldec Those types describe the messageBundle in which we represent the messages internally in.

Reply:
rename to message-bundle?